pkg/parser/pipelineyml/define.go

```diff
@@ -36,9 +38,9 @@ type Spec struct {
 
 	Name string `yaml:"name"`
 
-	On       *TriggerConfig                `yaml:"on,omitempty"`
-	Triggers []*apistructs.PipelineTrigger `yaml:"triggers,omitempty"` // todo Solve the problem that quotation marks will be automatically added after yaml is saved
```

Comment:
`apistructs.PipelineTrigger` should be deleted

Reply:
done